### PR TITLE
New tool: MITOS

### PR DESCRIPTION
**Pull request by bernt-matthias**
I would like to add MITOS -- a tool for the automatic de-novo annotation of metazoan mitochondrial genomes -- to the Galaxy tools. Currently MITOS runs here http://mitos.bioinf.uni-leipzig.de/ and has been made available via pypi and conda recently. 

Over the last years our webserver was quite successful (approx 2000 jobs resp 3000 visits / month) -- so I would also hope that this draws a bit more attention to the usegalaxy* servers.

The MITOS2 tool will be added soon .. the data manager is already prepared for this. 

One question: It would be nice if our support mail address would be included in the tool form. Is there a way to do this without drawing in even more spam? 

FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [x] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

**Comment by bgruening** [comment]
> t would be nice if our support mail address would be included in the tool form. Is there a way to do this without drawing in even more spam?

We do not have any formal way of doing this imho. But if this is important for you you can probably add this to the help text.

**Comment by bernt-matthias** [comment]
Left todo: 

- manual test of data manager + tools
- help for MITOS2

File: data_managers/data_manager_mitos/.shed.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+categories:
+- Data Managers
+description: MITOS data manager.
+homepage_url: ''
+long_description: 'Download reference data for MITOS and MITOS2'
+name: data_manager_mitos
+owner: iuc
+remote_repository_url: https://github.com/bernt-matthias/mb-galaxy-tools/tree/master/data_managers/data_manager_mitos
```
Comment (bgruening) [inline data_managers/data_manager_mitos/.shed.yml]: iuc?

File: data_managers/data_manager_mitos/.shed.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+categories:
+- Data Managers
+description: MITOS data manager.
+homepage_url: ''
```
Comment (bgruening) [inline data_managers/data_manager_mitos/.shed.yml]: mitos webpage?

File: tools/mitos/mitos2.xml [hunk 3]
```diff
@@ -0,0 +1,360 @@
...
+
+  Select the feature types that sould be annotated. By default this is protein coding genes, tRNA and rRNA which is useful for metazoan mitogenomes. In addition also the replication origins of the light (OL) and heavy (OH) strand and introns can be annotated. The annotation of the replication origins is most useful for chordate mitogenomes. Introns are ususally only found in mitogenomes of non-metazoans and basal metazoa.
+
+- Final overlap (nt)
+
+  Maximum number of nucleotides by which genes of different types may overlap. Applies to merging of the final predictions. 
+
+- Annotate only the best copy of each feature
+
+  TODO
```
Comment (bgruening) [inline tools/mitos/mitos2.xml]: ToDo :)

File: data_managers/data_manager_mitos/.shed.yml (hunk 1 above)
Comment (bernt-matthias) [reply data_managers/data_manager_mitos/.shed.yml]: done

File: data_managers/data_manager_mitos/.shed.yml (hunk 2 above)
Comment (bgruening) [reply data_managers/data_manager_mitos/.shed.yml]: ping!

**Comment by bernt-matthias** [comment]
Yeah. Ready. Not sure why the data manager fails

**Comment by bernt-matthias** [comment]
Done. One last point: test-data is ~10MB. Is this to much?

**Comment by bgruening** [comment]
Can you do it with smaller test data?

**Comment by bernt-matthias** [comment]
> Can you do it with smaller test data?

Think so... will ping if ready

**Comment by bernt-matthias** [comment]
Hey @bgruening: Ready from my side test-data is now 2.7MB - I have squashed all commits. 

Tests pass locally for me, no idea why tests are not started here...

**Comment by bgruening** [comment]
It turned green here: https://travis-ci.org/github/galaxyproject/tools-iuc/jobs/667832401

**Comment by bgruening** [comment]
Danke @bernt-matthias!

**Comment by bernt-matthias** [comment]
Cool. What is the procedure to get this on usegalaxy? I'm already thinking about a mini training .. but this will need a bit.

**Comment by bgruening** [comment]
Ha, just wrote you an email :)

**Pull request merged**